infoschema/perfschema/tables.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2017 PingCAP, Inc.
```

Comment:
```suggestion
// Copyright 2019 PingCAP, Inc.
```

Reply:
I'm not sure...
This code is moved, rather than new generated ~